packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -112,6 +112,34 @@
         <protected>false</protected>
         <shortDescription>Show less values for the potato facet</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_AllCategories</fullName>
+        <value>All Categories</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>All Categories</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_InLabel</fullName>
+        <value>in</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>in</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_Plus</fullName>
+        <value>+</value>
```

Comment:
I don't think `+` and `-` need to be localized or translated. Unless I'm just unaware that this has a different meaning depending on the language. But I believe it's a pretty standard math symbol.

Reply:
I guess it was modified this way because of [my comment](https://github.com/coveo/ui-kit/pull/1135#discussion_r700319402).

I agree with you that math symbols are pretty universal, but I can easily imagine a customer that would not want to show plus and minus signs. In my opinion, the plus/minus sign is only styling in this context. Maybe my comment was unclear, but what I meant was to include the plus/minus sign in the localized string. This way, to get rid of the plus/minus sign, a customer could simply override the localized string instead of having to modify the facet itself.

As you mentioned, I would not define separate localized strings for the plus/minus signs.

Reply:
Ah yes i totally agree with having the sign inside the label 👍